specification/iothub/resource-manager/Microsoft.Devices/preview/2019-03-22-preview/iothub.json

```diff
@@ -2045,6 +2045,35 @@
             "name": "Capabilities",
             "modelAsString": true
           }
+        },
+        "locations": {
```

Comment:
Is this the right way to model this concept? how do other Azure services do this? Can you provide some references, if you have? Should it be that primary is always defaulted to the location of the azure resource as provided by the customer and then the secondary is the closest fail over region?

Reply:
Please share some more details on how these role types are used in iot hubs space. Feel free to set up a quick call to close on this.

Reply:
As mentioned above these locations are used to determine the current location and the failover location of the resource. I have closed this PR and created a new one : https://github.com/Azure/azure-rest-api-specs/pull/6479 i have already added you to it. If you have further questions lets set up some time tomorrow.